README.md

```diff
@@ -62,6 +62,16 @@ const packageName = await prompt('Package name', {
 })
 ```
 
+**This package provide some validators for common usage**
+
+- required
+
+```js
+import { prompt, Validators } from "@topcli/prompts";
+
+const name = await prompt("What's your name ?", { validators: [Validators.required] });
```

Comment:
Jump line for the options? (that's quite a lot of code on the same line).

Reply:
I've also updated others snippets. Some snippets were written in `standard` way, it's also updated.